src/settings.plugin.js

```diff
@@ -157,33 +173,52 @@ wikiaJWPlayerSettingsPlugin.prototype.createSubmenuWrapper = function () {
 
 // autoplay button specific methods
 wikiaJWPlayerSettingsPlugin.prototype.createAutoplayToggle = function () {
-	var autoplayToggle = createToggle({
-			id: this.player.getContainer().id + '-videoAutoplayToggle',
-			label: this.config.i18n.autoplayVideos,
-			checked: this.config.autoplay
+	function toggleAutoplay(event) {
+		this.player.trigger('autoplayToggle', {
+			enabled: !event.target.previousSibling.checked
 		});
+	}
 
-	autoplayToggle.querySelector('label')
-		.addEventListener('click', function (event) {
-			this.player.trigger('autoplayToggle', {
-				enabled: !event.target.previousSibling.checked
-			});
-		}.bind(this));
+	var autoplayToggle = createToggle({
+		id: this.player.getContainer().id + '-videoAutoplayToggle',
+		label: this.config.i18n.autoplayVideos,
+		checked: this.config.autoplay
+	});
+
+	var label = autoplayToggle.querySelector('label');
+
+	label.addEventListener('keydown', function (evt) {
+		if ([13, 32].indexOf(evt.keyCode) !== -1) {
+			evt.preventDefault();
+			evt.stopPropagation();
+
+			var input = event.target.previousSibling;
+
+			input.checked = !input.checked;
+		}
+	});
+
+	this._onClick(label, toggleAutoplay);
```

Comment:
It adds second `keydown` listener. Do we need both?

Reply:
Good point!